packages/core-magistrate-transactions/src/handlers/bridgechain-update.ts

```diff
@@ -134,13 +136,8 @@ export class BridgechainUpdateTransactionHandler extends MagistrateTransactionHa
         const bridgechainAttributes: IBridgechainWalletAttributes =
             businessAttributes.bridgechains[bridgechainUpdate.bridgechainId];
 
-        if (bridgechainUpdate.seedNodes) {
-            bridgechainAttributes.bridgechainAsset.seedNodes = bridgechainUpdate.seedNodes;
-        }
-
-        if (bridgechainUpdate.ports) {
-            bridgechainAttributes.bridgechainAsset.ports = bridgechainUpdate.ports;
-        }
+        delete bridgechainUpdate.bridgechainId; // we don't want id in wallet bridgechain asset
```

Comment:
Instead of mutating the transaction asset, do it the other way around. Assign first and then delete the id from the attributes. It could introduce some unwanted side-effects.

Reply:
yes wasn't convinced about my approach either :smile: will update as you said, thanks